test/_autoload.php

```diff
@@ -1,9 +1,12 @@
 <?php
 
+declare(strict_types=1);
+
 use PHPUnit\Framework\Assert;
 
-if (! class_exists(PHPUnit_Framework_Assert::class)
+if (
+    ! class_exists(phpunit_framework_assert::class)
```

Comment:
This is something phpcs does when it doesn't recognize a class, due to it not being loaded. Which ... of course it isn't, but that's why the conditional is here. While this is likely safe (class names are technically case insensitive in the engine), this should probably be reverted, and an ignore rule setup for this file.

Reply:
done